samples/snippets/src/main/java/com/example/mediatranslation/TranslateFromMic.java

```diff
@@ -0,0 +1,133 @@
+
+              if (res.getTextTranslationResult().getIsFinal()) {
+                System.out.println(String.format("\nFinal translation: %s", translation));
+                System.out.println(String.format("Final recognition result: %s", source));
+              } else {
+                System.out.println(String.format("\nPartial translation: %s", translation));
+                System.out.println(String.format("Partial recognition result: %s", source));
+              }
+            }
+
+            @Override
+            public void onComplete() {}
+
+            public void onError(Throwable t) {
+              System.out.println(t);
+            }
+          };
+
+      ClientStream<StreamingTranslateSpeechRequest> clientStream =
+          client.streamingTranslateSpeechCallable().splitCall(responseObserver);
+
+      TranslateSpeechConfig audioConfig =
+          TranslateSpeechConfig.newBuilder()
+              .setAudioEncoding("linear16")
+              .setSourceLanguageCode("en-US")
+              .setTargetLanguageCode("es-ES")
+              .setSampleRateHertz(16000)
+              .build();
+
+      StreamingTranslateSpeechConfig streamingRecognitionConfig =
+          StreamingTranslateSpeechConfig.newBuilder().setAudioConfig(audioConfig).build();
+
+      StreamingTranslateSpeechRequest request =
+          StreamingTranslateSpeechRequest.newBuilder()
+              .setStreamingConfig(streamingRecognitionConfig)
+              .build(); // The first request in a streaming call has to be a config
+
+      clientStream.send(request);
+      // SampleRate:16000Hz, SampleSizeInBits: 16, Number of channels: 1, Signed: true,
+      // bigEndian: false
+      AudioFormat audioFormat = new AudioFormat(16000, 16, 1, true, false);
+      DataLine.Info targetInfo =
+          new DataLine.Info(
+              TargetDataLine.class,
+              audioFormat); // Set the system information to read from the microphone audio stream
+
+      if (!AudioSystem.isLineSupported(targetInfo)) {
+        System.out.println("Microphone not supported");
+        System.exit(0);
+      }
+      // Target data line captures the audio stream the microphone produces.
+      TargetDataLine targetDataLine = (TargetDataLine) AudioSystem.getLine(targetInfo);
+      targetDataLine.open(audioFormat);
+      targetDataLine.start();
+      System.out.println("Start speaking... Press Ctrl-C to stop");
+      long startTime = System.currentTimeMillis();
+      // Audio Input Stream
+      AudioInputStream audio = new AudioInputStream(targetDataLine);
+
+      while (true) {
+        byte[] data = new byte[6400];
+        audio.read(data);
+        request =
+            StreamingTranslateSpeechRequest.newBuilder()
+                .setAudioContent(ByteString.copyFrom(data))
+                .build();
+        clientStream.send(request);
+      }
+    } catch (Exception e) {
+      System.out.println(e);
+    }
```

Comment:
What's the catch for? 
If it just prints the exception, it would be preferable to just let the library throw it.